src/components/styles.js

```diff
@@ -75,6 +80,9 @@ export const ClauseBody = styled.div`
   .computed {
     color: ${props => props.computedcolor || '#f1baff'};
   }
+  li{
+    transition:none;
```

Comment:
I think I was mistaken, it must be `transition-duration` because they still don't instantly change position.

Reply:
@irmerk I just tried it locally and it doesn't still make any difference. The "transition" is a shorthand to setting  "<property> <duration> <timing-function> <delay>", so setting the transition to none takes care of all the transition effects. I think this works though because if you adding or just removing the last element, it changes position instantly. The transition effect seems to happen when an element from any other position is being removed, of which I think it makes sense for it to work that way, so as to smoothly cover up the position of the element. So it seems like a default behavior and I'm not sure what to do to override it.